sources/au/victoria.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "http",
-    "data": "http://s3.amazonaws.com/data.openaddresses.io/cache/au-victoria.zip",
-    "website": "http://services.land.vic.gov.au/landchannel/content/vicmapdata?productID=1",
+    "data": "https://www.alantgeo.com.au/share/au-victoria.zip",
```

Comment:
This URL is 404'ing. Can you throw it in Dropbox or similar and let us know what the URL is so we can add it to our S3 bucket?